src/GraphQL.NewtonsoftJson/StringExtensions.cs

```diff
@@ -26,12 +24,12 @@ public static Inputs ToInputs(this string json)
         public static Inputs ToInputs(this JObject obj)
         {
             var variables = obj?.GetValue() as Dictionary<string, object>
-                            ?? new Dictionary<string, object>();
+                ?? new Dictionary<string, object>();
             return new Inputs(variables);
```

Comment:
```c#
var variables = obj?.GetValue() as Dictionary<string, object>;
return variables.ToInputs();
```

Reply:
One less allocation. Nice. I've also added `remarks` to let anyone know this method is used by the server project. It could potentially be removed from here entirely, since we don't use it anymore, but I guess some folks could be using it in their own projects.